### DISCUSSION TITLE
Review Spam Take Down! Please help!

### DISCUSSION THREAD

**Discussion by gnonymousgems** [community/community#72244] [Pull Requests]
I have this another account @gaurovgiri from which spam reviews were created on 10/24/2023 while working on an automating script. Can I take down those reviews and make it not count in that profile?

**Comment by Fullovka** [comment]
To solve the issue with spam reviews created from your GitHub account and remove or prevent their impact on your profile, follow these steps:

1. Identify the spam reviews: first, identify the specific repositories where the spam reviews were created on October 24, 2023.

2. Delete the spam reviews: if you have access to the repositories where the spam reviews were created, you can delete them manually by following these steps:

- 2.1 Log in to your GitHub account: open a web browser and go to the GitHub website. Log in to your account using your credentials.

- 2.2 Navigate to the repository: find and navigate to the repository where the spam reviews were created. You can use the GitHub search feature to locate the specific repository.

- 2.3 Open the "Issues" or "Pull Requests" section: inside the repository, go to the "Issues" (if the spam reviews were created as issues) or "Pull Requests" (if the spam reviews were created as pull requests) section.

- 2.4 Locate the spam reviews: scroll through the list of issues or pull requests to find the spam reviews. They typically contain unwanted comments or reviews.

- 2.5 Delete the spam reviews: open the spam reviews and find the option to delete the comments or reviews. This option is usually labeled as "Delete." Click on this option to delete the spam reviews.

- 2.6 Confirm the deletion: you may need to confirm the deletion of the comments. Confirm that you want to delete the spam reviews.

- 2.7 Repeat for other spam reviews: if you have multiple spam reviews, repeat the deletion process for each of them.

- 2.8 Verify that the spam reviews are removed: ensure that all spam reviews have been successfully deleted from the repository.

3. Contact repository owners/admins: if you do not have access to the repositories where the spam reviews were created, contact the owners or administrators of those repositories and ask them to remove the spam reviews. Explain the situation and provide any necessary additional information.

4. Avoid spamming: make sure that you do not engage in spam activities on GitHub or any other platforms. It is important to use the platform responsibly and avoid actions that violate GitHub's terms of service.

5. Adjust your automated script: if the spam reviews were a result of your automated script, review and modify your script to prevent such actions in the future. Ensure your script complies with GitHub's guidelines and policies.

6. Contact GitHub support: if you have taken the necessary actions to remove the spam reviews but they still appear on your profile, you can contact GitHub support and explain the situation to them. They may be able to provide further assistance in resolving the issue.

Remember that GitHub has policies against spam and misuse of the platform, so it's essential to maintain good behavior and follow their guidelines to avoid any further issues with your account.
